sql/analyzer/validation_rules.go

```diff
@@ -80,7 +80,7 @@ var (
 	// ErrSubqueryMultipleColumns is returned when an expression subquery returns
 	// more than a single column.
 	ErrSubqueryMultipleColumns = errors.NewKind(
-		"subquery expressions can only return a single column",
+		"operand contains more than one column",
```

Comment:
I think we should create a new package for errors. That way errors like these can be properly associated with their MySQL code.